packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -559,11 +570,15 @@ class RenderEditable extends RenderBox {
 
   Offset _lastTapDownPosition;
   Offset _longPressPosition;
```

Comment:
i'm pretty sure we can simplify this by removing _longPressPosition and never setting _lastTapDownPosition to null. It doesn't seem we really depend on any of this.

Reply:
That sounds reasonable; I will submit a separate patch for that.